anitya/lib/backends/maven.py

```diff
@@ -70,3 +75,34 @@ def get_versions(cls, project):
                           .replace(':', '/'))
 
         return get_versions_by_regex(url, REGEX, project)
+
+    @classmethod
+    def check_feed(cls):
+        ''' Return a generator over the latest 40 packages to Maven central index
+
+        by calling maven-index-checker application
+        '''
+        maven_url = 'http://repo2.maven.org/maven2'
+        jar_path = '/src/maven-release-checker-1.0-SNAPSHOT-jar-with-dependencies.jar'
+        # session for checking whether there is already created package
+        session = init(APP.config['DB_URL'], create=True, debug=False)
+
+        try:
+            data = subprocess.check_output(["java", "-jar", jar_path], universal_newlines=True)
+        except Exception:
+            raise AnityaPluginException('Could not start %s' % maven_url)
+
+        data = json.loads(data)
+        for item in data[:40]:
+            item = json.loads(item)
+            version_url = '{group_id}:{artifact_id}'.format(group_id=item['groupId'], artifact_id=item['artifactId'])
+            retrieved_project = Project.by_version_url(session, version_url)
```

Comment:
Yeah maybe we can change the variable name and put a comment that for maven backend we store the coordinate in the version_url field